package.json

```diff
@@ -14,6 +14,7 @@
     "cloud-vanilla-theme": "0.1.2",
     "cryptojs": ">= 2.5.3",
     "graceful-fs": "4.1.11",
+    "js-yaml": "https://registry.npmjs.org/js-yaml/-/js-yaml-3.9.1.tgz",
```

Comment:
the static asset for js-yaml was removed and it's now installed via npm and will be required in.